Dockerfile

```diff
@@ -36,6 +36,11 @@ RUN apt update && \
     bundle config set --local without 'development test' && \
     bundle config set silence_root_warning true && \
     bundle install -j"$(nproc)" && \
+    # don't need .gem files once installed, but we do need to keep the git cloned cache
+    rm -rf /opt/mastodon/vendor/bundle/ruby/3.0.0/cache/*.gem && \
```

Comment:
Interesting that this is the path given that I thought we were using 3.0.4...

Do you know if this path will change with https://github.com/mastodon/mastodon/pull/21061 ?

Reply:
It will. Rubygems uses `major.minor.0` as path separators as far as I remember.

Reply:
iirc, it can be more nuanced than that (I remember much older versions having the same for awhile), but yeah in general `major.minor.0` as @moritzheiber said.

If the Ruby version was an ARG or ENV, then we could use it here instead of hard-coding it.

Reply:
You can also use `*` instead of the version component in the path, you don't have to be 100% precise.